src/main/resources/worldofzuul.presentation/house.fxml

```diff
@@ -11,43 +11,64 @@
    <children>
       <ImageView fx:id="houseImage" fitHeight="400.0" fitWidth="600.0" pickOnBounds="true">
          <image>
-            <Image url="@../images/SemesterProjektMapNY.png" />
+            <Image url="@../images/SemesterProjektMap5Huse.png" />
          </image>
       </ImageView>
-      <Label layoutX="311.0" layoutY="67.0" text="House">
+      <Label layoutX="310.0" layoutY="4.0" text="House">
          <font>
             <Font size="14.0" />
          </font></Label>
       <Button fx:id="btnNextYear" layoutX="14.0" layoutY="14.0" mnemonicParsing="false" onAction="#handleBtnNextYear" text="Next Year" />
       <Button fx:id="btnHelp" layoutX="100.0" layoutY="14.0" mnemonicParsing="false" onAction="#handleBtnHelp" text="Help" />
-      <Label fx:id="labelBuildArea" layoutX="68.0" layoutY="235.0" text="Build Area" onMouseClicked="#handleLabelBuildArea">
+      <Label fx:id="labelBuildArea" layoutX="68.0" layoutY="235.0" onMouseClicked="#handleLabelBuildArea" text="Build Area">
```

Comment:
I think we also need to add so that the player can click on the build area itself, not just the label.